app/components/Content/Widget.js

```diff
@@ -25,15 +25,15 @@ const StyledCard = styled(Card).attrs({
   width: 100%;
   margin-bottom: 30px;
   position: relative;
-  padding: 24px;
+  padding: ${2 * GU}px;
```

Comment:
`24px` equals `3 * GU` are we sure we want to reduce the padding from the designs?, I would confirm first with Javi or Yalda